app/javascript/articles/components/CommentsCount.jsx

```diff
@@ -1,10 +1,12 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 
-export const CommentsCount = ({ count, articlePath, icon }) => {
+export const CommentsCount = ({ count, articlePath, icon, className }) => {
   if (count > 0) {
     return (
-      <div className="article-engagement-count comments-count">
+      <div
+        className={`article-engagement-count comments-count ${className || ''}`}
```

Comment:
This is to support the `<FeaturedArticle />` component which takes additional "featured" classes.